lib/Typesense/Documents.d.ts

```diff
@@ -70,6 +70,9 @@ export interface SearchResponseHit<T extends DocumentSchema> {
             matched_tokens: string[];
         }
     ];
+    geo_distance_meters?: {
+		location?: number;
```

Comment:
`location` is actually the name of the `geopoint` field from the document. So this needs to be a dynamic type which allows keys with the same name as any of the document's fields...